src/components/Episodes/data.js

```diff
@@ -1,22 +1,22 @@
 export default [
 	{
-		number: 1,
-		image: {
+		episodeNo: 1,
```

Comment:
the values here should still be `number` not `episodeNo` - it's in a data object that represents an array of episodes, so it's clear that the number would be the episode number; `episodeNo` is just what we can name them during the extraction process or whatever.